### PR TITLE
Rename from Rocketlets to Rocket.Chat Apps

### PR DESCRIPTION
**Pull request by graywolf336**

### CODE REVIEW

File: example-app.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+    //This is an example of how a app.json file will look like
     "name": "Testing",
     "nameSlug": "testing",
     "description": "Testing description",
     "version": "1.0.0",
     "requiredApiVersion": "0.0.1",
     "author": {
         "name": "Bradley Hilton",
-        "support": "https://github.com/graywolf336/temporary-rocketlets-ts-definition/"
+        "support": "https://github.com/graywolf336/temporary-apps-ts-definition/"
```
Comment (geekgonecrazy) [inline example-app.json]: missed one :)

File: package.json [hunk 2]
```diff
@@ -10,17 +10,17 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/graywolf336/temporary-rocketlets-ts-definition.git"
+    "url": "git+https://github.com/graywolf336/temporary-apps-ts-definition.git"
```
Comment (geekgonecrazy) [inline package.json]: :top:

File: package.json [hunk 3]
```diff
@@ -10,17 +10,17 @@
...
   },
   "keywords": [
-    "rocketlets"
+    "apps"
   ],
   "author": "Bradley Hilton <bradley.hilton@rocket.chat>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/graywolf336/temporary-rocketlets-ts-definition/issues"
+    "url": "https://github.com/graywolf336/temporary-apps-ts-definition/issues"
```
Comment (geekgonecrazy) [inline package.json]: :top:  ok i'll stop :grin:

Comment (graywolf336) [reply package.json]: The repository wasn't transferred when this commit was done ;)

**Pull request merged**